screens/Settings/SettingScreen.tsx

```diff
@@ -0,0 +1,165 @@
+            color={Theme.Colors.Icon}
+            containerStyle={Theme.Styles.settingsIconBg}
+          />
+          <ListItem.Content>
+            <ListItem.Title>
+              <Text weight="semibold">{t('language')}</Text>
+            </ListItem.Title>
+          </ListItem.Content>
+          <Icon
+            name="chevron-right"
+            size={21}
+            color={Theme.Colors.profileLanguageValue}
+            style={{ marginRight: 15 }}
+          />
+        </ListItem>
+      }
+    />
+  );
+};
+
+let helpUrl = '';
+
+let helpPage = getAllConfigurations().then((response) => {
+  helpUrl = response.helpUrl;
+});
+
+export const SettingScreen: React.FC<SettingProps & MainRouteProps> = (
+  props
+) => {
+  const { t } = useTranslation('SettingScreen');
+  const controller = useSettingsScreen(props);
+
+  return (
+    <React.Fragment>
+      <Pressable onPress={controller.TOGGLE_SETTINGS}>
+        {props.triggerComponent}
+      </Pressable>
+      <Modal
+        isVisible={controller.isVisible}
+        arrowLeft={<Icon name={''} />}
+        headerTitle={t('header')}
+        headerElevation={2}
+        onDismiss={controller.TOGGLE_SETTINGS}>
+        <ScrollView backgroundColor={Theme.Colors.lightGreyBackgroundColor}>
+          <Column fill>
+            <MessageOverlay
+              isVisible={controller.alertMsg != ''}
+              onBackdropPress={controller.hideAlert}
+              title={controller.alertMsg}
+            />
+
+            <LanguageSetting />
+
+            <ListItem topDivider disabled={!controller.canUseBiometrics}>
+              <Image
+                source={require('../../assets/biometric-unlock-icon.png')}
+                style={{ marginLeft: 10, marginRight: 9 }}
+              />
+              <ListItem.Content>
+                <ListItem.Title>
+                  <Text weight="semibold" color={Theme.Colors.profileLabel}>
+                    {t('bioUnlock')}
+                  </Text>
+                </ListItem.Title>
+              </ListItem.Content>
+              <Switch
+                value={controller.isBiometricUnlockEnabled}
+                onValueChange={controller.useBiometrics}
+                color={Theme.Colors.profileValue}
+              />
+            </ListItem>
+
+            <AboutInji />
+
+            <ListItem
+              topDivider
+              bottomDivider
+              onPress={() => {
+                Linking.openURL(helpUrl);
+              }}>
+              <Image
+                source={require('../../assets/features-walkaround-icon.png')}
+                style={{ marginLeft: 10, marginRight: 9 }}
+              />
+              <ListItem.Content>
+                <ListItem.Title>
+                  <Text weight="semibold" color={Theme.Colors.profileLabel}>
+                    {t('injiTourGuide')}
+                  </Text>
+                </ListItem.Title>
+              </ListItem.Content>
+            </ListItem>
+
```

Comment:
in the new settings screen, we are not able to see the credential registry edit implementation. please refer old ProfileScreen.tsx.

Reply:
Implemented Credential registry edit in the new setting screen.